src/main/scala/wdl4s/values/WdlFile.scala

```diff
@@ -35,14 +39,20 @@ sealed trait WdlFile extends WdlPrimitive {
     case _ => invalid(s"$value == $rhs")
   }
 
-  override def valueString = value.toString
+  override def valueString: String = value.toString
 }
 
-case class WdlSingleFile(value: String) extends WdlFile {
-  override def toWdlString = "\"" + value.toString + "\""
+case class WdlSingleFile(origValue: String) extends WdlFile {
+  override def toWdlString: String = "\"" + value.toString + "\""
+
+  // Validate file path
```

Comment:
You're not really taking advantage of Refined here, at least I don't think so. 

The power of that library is to define a type which has been refined further, e.g. "an integer larger than 5", or "a string which is not empty". You're not actually doing that here.